src/components/clusterWizard/ClusterWizardForm.tsx

```diff
@@ -89,63 +103,131 @@ const ClusterWizardForm: React.FC<ClusterWizardFormProps> = ({ cluster, setStep
+                      isRequired
+                    />
+                    <InputField
+                      name="serviceNetworkCIDR"
+                      label="Service Network CIDR"
+                      helperText="The IP address pool to use for service IP addresses. You can enter only one IP address pool. If you need to access the services from an external network, configure load balancers and routers to manage the traffic."
+                      isRequired
+                    />
                     <InputField
                       label="API Virtual IP"
-                      name="apiVIP"
-                      helperText="This can not be changed after cluster is deployed"
+                      name="apiVip"
+                      helperText="Virtual IP used to reach the OpenShift cluster API."
+                      isRequired
+                    />
+                    <InputField
+                      name="dnsVip"
+                      label="Internal DNS Virtual IP"
+                      helperText="Virtual IP used internally by the cluster for automating internal DNS requirements."
+                      isRequired
+                    />
+                    <InputField
+                      name="ingressVip"
+                      label="Ingress Virtual IP"
+                      helperText="Virtual IP used for cluster ingress traffic."
                       isRequired
                     />
-                    <InputField name="dnsVIP" label="Internal DNS Virtual IP" isRequired />
-                    <InputField name="ingressVIP" label="Ingress Virtual IP" isRequired />
-                    <InputField name="machineCIDR" label="Machine CIDR" placeholder="" isRequired />
                     <TextContent>
                       <Text component="h2">Security</Text>
                     </TextContent>
-                    <TextAreaField name="pullSecret" label="Pull Secret" isRequired />
-                    <TextAreaField name="sshPrivateKey" label="SSH Private Key" isRequired />
-                    <TextAreaField name="sshPublicKey" label="SSH Public Key" isRequired />
+                    <TextAreaField
+                      name="pullSecret"
+                      label="Pull Secret"
+                      helperText={
+                        <>
+                          The pull secret obtained from the Pull Secret page on the{' '}
+                          {
+                            <a
+                              href="https://cloud.redhat.com/openshift/install/metal/user-provisioned"
```

Comment:
A constant again. I think we should introduce a file with just these "config-like" constants.